### PR TITLE
docs(natives/PED): Change documentation SET_PED_CONFIG_FLAGS

### PR DESCRIPTION
**Pull request by BugCode1**
Adding a complete list of flags with string names

### CODE REVIEW

**Comment by AvarianKnight** [comment]
Deleting the entire enum seems a bit weird, you should just update the current enums on the ones that changed.

Also please read the [contributor guidelines](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md#external-links), you shouldn't link to external websites as the links get deleted

**Comment by BugCode1** [comment]
There are too many of them and they don't fit properly. To read them, you have to scroll horizontally and vertically or zoom out the browser.

On the other hand, many links of the pastebin website are in the documentation of some natives.

For example, the:
https://docs.fivem.net/natives/?_0x8D8686B622B88120

However, if you say, I will put the whole list in the form of an array as before.

**Comment by AvarianKnight** [comment]
> There are too many of them and they don't fit properly. To read them, you have to scroll horizontally and vertically or zoom out the browser.

A lot of the ones in the list are already documented and are known true (any with CPED_CONFIG_FLAG_[name] are hash matches iirc, you'' notice some that have comments beside them because they likely don't match)

> On the other hand, many links of the pastebin website are in the documentation of some natives.
> For example, the: [docs.fivem.net/natives/?_0x8D8686B622B88120](https://docs.fivem.net/natives/?_0x8D8686B622B88120)

The doc you listed are from when FiveM did their original Natives import, any that link to outside websites *should* be replaced with in-line docs or link to the citizenfx/fivem-docs

**Comment by coalaura** [comment]
Idk how accurate this source is but there seem to be a few more named ones [here](https://alexguirre.github.io/rage-parser-dumps/dump.html?game=gta5&build=2944&search=config#ePedConfigFlags)

**Comment by BugCode1** [comment]
> Idk how accurate this source is but there seem to be a few more named ones [here](https://alexguirre.github.io/rage-parser-dumps/dump.html?game=gta5&build=2944&search=config#ePedConfigFlags)

The source you mentioned was better and had more flags that I added. Thanks

File: PED/SetPedConfigFlag.md [hunk 1]
```diff
@@ -110,370 +109,377 @@ enum ePedConfigFlags {
...
+	CPED_CONFIG_FLAG_DontCryForHelpOnStun = 454,
 	_0xB68D3EAB = 455,
 	CPED_CONFIG_FLAG_CanBeIncapacitated = 456,
 	_0x4BD5EBAD = 457,
+	CPED_CONFIG_FLAG_DontChangeTargetFromMelee = 458,
+	_0x4376ABF2 = 459,
+	CPED_CONFIG_FLAG_RagdollFloatsIndefinitely = 460,
+	CPED_CONFIG_FLAG_BlockElectricWeaponDamage = 461,
+	_0x262A3B8E = 462,
+	_0x1AA79A25 = 463
```
Comment (PsychoShock) [inline PED/SetPedConfigFlag.md:476]: Not sure about that. I can't seem to found aything about that.

**Review by PsychoShock — APPROVED** [review]
I'll PR the rest missing flag when this if get merged. I check them all and seem's the be all valid.

**Comment by coalaura** [comment]
This PR seems to have the full list #1082

**Comment by AvarianKnight** [comment]
Superseded by https://github.com/citizenfx/natives/pull/1082

**Pull request closed without merging**